ui/app/mirrors/[mirrorId]/cdcDetails.tsx

```diff
@@ -151,8 +148,7 @@ const SyncIntervalLabel: React.FC<{ syncInterval?: number }> = ({
 
   if (!syncInterval) {
     return <ProgressCircle variant='determinate_progress_circle' />;
-  }
-  if (syncInterval >= 3600) {
+  } else if (syncInterval >= 3600) {
```

Comment:
unrelated but ideally this should be hours + minutes in case it's not exactly an hour